Surface_mesh_skeletonization/include/CGAL/Mean_curvature_flow_skeletonization.h

```diff
@@ -1320,12 +1322,22 @@ std::size_t collapse_short_edges();
   /// Compute an approximate vertex normal for all vertices.
   void compute_vertex_normal()
   {
+    namespace PMP = CGAL::Polygon_mesh_processing;
+
+    boost::unordered_map<face_descriptor, Vector> normals;
+    boost::associative_property_map<
+      boost::unordered_map<face_descriptor, Vector> > normals_pmap(normals);
+    PMP::compute_face_normals(m_tmesh, normals_pmap);
+
```

Comment:
IIRC, the polyhedron type used has ids in the faces, using a `std::vector` for storing the face normal would be more efficient. The vertex normal are recomputed at each contraction step so I guess this should make a difference.